lib/scaffold.dart

```diff
@@ -707,12 +714,12 @@ class BackdropScaffoldState extends State<BackdropScaffold>
 /// Credit: https://stackoverflow.com/a/60868972/2554745
 class _MeasureSize extends StatefulWidget {
   final Widget child;
-  final ValueChanged<Size> onChange;
+  final ValueChanged<Size?> onChange;
```

Comment:
Can't do anything to change `Size?` to `Size` here, right?

Reply:
Revert back to `ValueChanged<Size>` in https://github.com/fluttercommunity/backdrop/pull/86/commits/2f4010e5619ad2089290835c3e40b7048594943f.

I have not studied "null safety" completely yet but added a null check before calling the "callback" should work.

Resolve this if you think that would work.